pycmr/granule.py

```diff
@@ -15,6 +16,12 @@ class GranuleQuery(object):
     def __init__(self):
         pass
 
+    def urlEncodeString(self, input):
+        """
+        Returns a URL-Encoded version of the given input parameter
+        """
+        return urllib.parse.quote(input)
```

Comment:
And then here you would use the function in question directly as opposed to traversing through the module structure.